Makefile

```diff
@@ -335,21 +325,22 @@ lint-code: $(GOLANGCI_LINT)
 lint: lint-code lint-actions lint-api lint-protos
 	@printf $(COLOR) "Run linters..."
 
-lint-api: $(API_LINTER)
+lint-api: $(API_LINTER) $(API_BINPB)
 	@printf $(COLOR) "Linting proto API..."
-	$(call silent_exec, $(API_LINTER) --set-exit-status $(PROTO_IMPORTS) --config=$(PROTO_ROOT)/api-linter.yaml $(PROTO_FILES))
+	$(call silent_exec, $(API_LINTER) --set-exit-status -I=$(PROTO_ROOT)/internal --descriptor-set-in $(API_BINPB) --config=$(PROTO_ROOT)/api-linter.yaml $(PROTO_FILES))
 
-lint-protos: $(BUF)
+lint-protos: $(BUF) $(API_BINPB)
 	@printf $(COLOR) "Linting proto definitions..."
-	@(cd $(PROTO_ROOT) && $(ROOT)/$(BUF) lint)
-
-buf-build: $(BUF)
-	@printf $(COLOR) "Build image.bin with buf..."
-	@(cd $(PROTO_ROOT) && $(ROOT)/$(BUF) build -o image.bin)
-
-buf-breaking: $(BUF)
-	@printf $(COLOR) "Run buf breaking changes check against image.bin..."
-	@(cd $(PROTO_ROOT) && $(ROOT)/$(BUF) check breaking --against image.bin)
+	@protoc --descriptor_set_in=$(API_BINPB) -I=proto/internal $(PROTO_FILES) -o /dev/stdout | (cd proto/internal && $(ROOT)/$(BUF) lint -)
+
+# TODO: fix this to work with getproto + API_BINPB
+# buf-build: $(BUF)
+# 	@printf $(COLOR) "Build image.bin with buf..."
+# 	@(cd $(PROTO_ROOT) && $(ROOT)/$(BUF) build -o image.bin)
+#
+# buf-breaking: $(BUF)
+# 	@printf $(COLOR) "Run buf breaking changes check against image.bin..."
+# 	@(cd $(PROTO_ROOT) && $(ROOT)/$(BUF) breaking --against image.bin)
```

Comment:
Do we rely on these for CI? I'd at least like the breaking changes check to work if at all possible

Reply:
They're unused as far as I can tell. image.bin is ignored in .gitignore which makes it kind of useless. If we did want to do this (which we do)... we should probably use buf's git support instead to check across branches. An in-repo file only allows linear checks so doesn't prevent cross-branch problems.